nservicebus/licensing/index_validity_core_[5,).partial.md

```diff
@@ -1,3 +1,3 @@
-The license is valid if the `ExpirationDate` or the `UpgradeProtectionExpiration` attribute in the license data is greater than or equal to the release date of the `{major}.{minor}.0` version of the NServiceBus assembly used by the endpoint. The release dates for the various versions can be found on [NuGet](https://www.nuget.org/packages/nservicebus).
+The license is valid if the `ExpirationDate` or the `UpgradeProtectionExpiration` attribute in the license data is greater than or equal to the release date of the `{major}.{minor}.0` version of the NServiceBus assembly used by the endpoint. To view the release dates for the various versions, see [NServiceBus Packages Versions](/nservicebus/upgrades/all-versions.md).
```

Comment:
No strong opinion, think we added this for our own internal knowledge as well.

>Having that information they can easily modify the license ;) not that they can't figure it out anyway, but aren't we kind of provoking that behavior?

There is no way to tamper with the license file so I don't think this applies